src/Centreon/Application/Webservice/TopologyWebservice.php

```diff
@@ -188,10 +189,39 @@ public function getNavigationList()
             ->getRepository(TopologyRepository::class)
             ->getTopologyList($user);
 
+        if ($pollerWizardEntity = $this->determinePollerWizardAccess($user)) {
+            $dbResult[] = $pollerWizardEntity;
+        }
+
         $status = true;
         $navConfig = $this->getDi()[ServiceProvider::YML_CONFIG]['navigation'];
         $result = new NavigationList($dbResult, $navConfig);
 
         return new Response($result, $status);
     }
+
+    /**
+     * @param \CentreonUser $user
+     * @return Topology|null
+     */
+    private function determinePollerWizardAccess(\CentreonUser $user): ?Topology
+    {
+        $userTopologyAccess = $user->access->getTopology();
+        if (
+            ! isset($userTopologyAccess[60901])
+            || (int) $userTopologyAccess[60901] !== \CentreonACL::ACL_ACCESS_READ_WRITE
+        ) {
+            return null;
+        }
+
+        $topology = new Topology();
+        $topology->setTopologyUrl("/poller-wizard/1");
+        $topology->setTopologyPage('60959');
+        $topology->setTopologyParent('60901');
```

Comment:
```suggestion
        $topology->setTopologyParent(60901);
```
setTopologyParent takes ?int as parameter

Reply:
I think you also could store 60901 in a variable as it is used 3 time in the method